### DISCUSSION TITLE
descending/

### DISCUSSION THREAD

**Discussion by giscus** [zmike/zmike.github.io#42] [Announcements]
# descending/

Into The Spiral of Madness

https://www.supergoodcode.com/descending/

**Comment by ET3D-ET3D** [comment]
Although I don't much like having to log in to comment, I just couldn't ignore the tremendous work that went into this post (and perhaps also into the code changes, but that's obviously not as important here at SGC).

Although I mourn the scarcity of the blog entries, I think that this particular one will take me at least until May to fully read, which somewhat alleviates the concern.

Who am I kidding? At this point I'm training a DNN (code named meme-machine) based on SGC so that I could get at least a semblance of Zink-related posts with stuff that I hardly understand but find very interesting all the same.

**Comment by konradmb** [comment]
> I was sitting at my battlestation doing some very ordinary **REDACTED** work for **REDACTED**

That one is easy - it's "some very ordinary **Half-Life 3** work for **Valve**"
